src/table/handle-scroll.js

```diff
@@ -0,0 +1,12 @@
+const handleScroll = (evt, tableSection) => {
+  evt.stopPropagation();
+  let { scrollLeft } = tableSection.current;
+  const { deltaX } = evt;
+  const max = tableSection.current.scrollWidth - tableSection.current.offsetWidth;
+  if (max > 0 && (scrollLeft + deltaX < 0 || scrollLeft + deltaX > max)) {
+    evt.preventDefault();
+    scrollLeft = Math.max(0, Math.min(max, scrollLeft + deltaX));
```

Comment:
I meant like this
```suggestion
  const newScrollLeft = tableSection.current.scrollLeft + evt.deltaX;
  const max = tableSection.current.scrollWidth - tableSection.current.offsetWidth;
  if (max > 0 && (newScrollLeft < 0 || newScrollLeft > max)) {
    evt.preventDefault();
    scrollLeft = Math.max(0, Math.min(max, newScrollLeft));
```

Reply:
scrollLeft in line 7 does not seem correct